### PR TITLE
WIP: arch/TMS320C64x: fix underflow

### PR DESCRIPTION
**Pull request by ret2libc**
(patch coming from radare2)

### CODE REVIEW

File: arch/TMS320C64x/TMS320C64xInstPrinter.c [hunk 1]
```diff
@@ -70,15 +70,14 @@ void TMS320C64x_post_printer(csh ud, cs_insn *insn, char *insn_asm, MCInst *mci)
...
-            if(p2 == p) {
-                strcpy(insn_asm, "Invalid!");
-                return;
-            } else {
-			    if(*p2 == 'A')
-				    strcpy(tmp, "1T");
-			    else
-				    strcpy(tmp, "2T");
-            }
+			if(p2 <= p) {
```
Comment (aquynh) [inline arch/TMS320C64x/TMS320C64xInstPrinter.c]: please put a space between `if` and `(`.

now i can see this file has this code style in many places, please help to fix them all, thanks!

Comment (ret2libc) [reply arch/TMS320C64x/TMS320C64xInstPrinter.c]: ack

**Comment by ret2libc** [comment]
spaces fixed

**Comment by aquynh** [comment]
please provide an input triggering this issue.

**Comment by ret2libc** [comment]
I have no such input :(

I think the case `p2 < p` should never happen, but it doesn't seem to harm either. I can keep the `<=` or switch back to `==` if you prefer, and just submit the style fixes.

**Comment by aquynh** [comment]
yes, please switch to `==` condition, and i will merge this.

**Comment by ret2libc** [comment]
done

**Comment by aquynh** [comment]
@fotisl, this is just cosmetic change, so i will merge. thanks!

**Pull request merged**